lesson3/hometask.rb

```diff
@@ -0,0 +1,21 @@
+module Myeach
+  def my_each
+    i = 0
+    while i < self.size
+      yield(self[i])
+      i+=1
+    end
+    self
+  end
+end
+
+class Array
+  include Myeach
+end
+
+class Hash
+  include Myeach
+end
+
+[1,2,3].my_each{|i|p i}
```

Comment:
check [this comment](https://github.com/nonkor/Betlab-ruby-courses/pull/13#discussion_r77856447), it's related to your work too